.run/ApiGatewayApplication.run.xml

```diff
@@ -11,7 +11,7 @@
       <option name="IS_ENABLE_EXPERIMENTAL_INTEGRATIONS" value="false" />
       <ENTRIES>
         <ENTRY IS_ENABLED="true" PARSER="runconfig" IS_EXECUTABLE="false" />
-        <ENTRY IS_ENABLED="true" PARSER="env" IS_EXECUTABLE="false" PATH="stack/local-docker.env" />
+        <ENTRY IS_ENABLED="true" PARSER="env" IS_EXECUTABLE="false" PATH="$PROJECT_DIR$/../digiwf-core/stack/local-docker.env" />
```

Comment:
Das scheint erst durch deinen Merge gekommen zu sein. Hat das einen Grund? Bei mir ging es immer ohne Probleme und wird sogar rückgängig gemacht beim starten.

Reply:
🤷‍♂️

![image](https://github.com/it-at-m/digiwf-core/assets/10849776/b92cef5b-0e4a-406e-a35a-1d88053095f4)

Reply:
Hab auf jeden Fall das gleiche Plugin in der gleichen Version.
Dumme Frage, aber du arbeitest schon direkt im digiwf-core Ordner und nicht z.B. in einem Ordner drüber?

Reply:
Keine dumme Frage, sondern das ist es🤦‍♂️

Ich hab ein IntelliJ "Multi-Projekt", sodass ich neben core, auch ops usw. alles drin hab. Sorry, ich ändere das wieder